codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/swiftmodules/AWSClientRuntimeTypes.kt

```diff
@@ -3,69 +3,53 @@ package software.amazon.smithy.aws.swift.codegen.swiftmodules
-        val AWSJSONError = runtimeSymbol("AWSJSONError")
-        val XAmzTargetMiddleware = runtimeSymbol("XAmzTargetMiddleware")
+        val AWSJSONError = runtimeSymbol("AWSJSONError", SwiftDeclaration.STRUCT)
+        val XAmzTargetMiddleware = runtimeSymbol("XAmzTargetMiddleware", SwiftDeclaration.STRUCT)
     }
     object RestJSON {
-        val RestJSONError = runtimeSymbol("RestJSONError")
+        val RestJSONError = runtimeSymbol("RestJSONError", SwiftDeclaration.STRUCT)
     }
 
     object RestXML {
-        val RestXMLError = runtimeSymbol("RestXMLError")
+        val RestXMLError = runtimeSymbol("RestXMLError", SwiftDeclaration.STRUCT)
         object S3 {
-            val AWSS3ServiceError = runtimeSymbol("AWSS3ServiceError")
-            val AWSS3ErrorWith200StatusXMLMiddleware = runtimeSymbol("AWSS3ErrorWith200StatusXMLMiddleware")
+            val AWSS3ServiceError = runtimeSymbol("AWSS3ServiceError", SwiftDeclaration.PROTOCOL)
+            val AWSS3ErrorWith200StatusXMLMiddleware = runtimeSymbol("AWSS3ErrorWith200StatusXMLMiddleware", SwiftDeclaration.STRUCT)
         }
     }
 
     object Core {
-        val AWSUserAgentMetadata = runtimeSymbol("AWSUserAgentMetadata")
-        val UserAgentMiddleware = runtimeSymbol("UserAgentMiddleware")
+        val AWSUserAgentMetadata = runtimeSymbol("AWSUserAgentMetadata", SwiftDeclaration.STRUCT)
+        val UserAgentMiddleware = runtimeSymbol("UserAgentMiddleware", SwiftDeclaration.STRUCT)
         val EndpointResolverMiddleware = runtimeSymbol("EndpointResolverMiddleware", SwiftDeclaration.STRUCT)
-        val FrameworkMetadata = runtimeSymbol("FrameworkMetadata")
-        val AWSClientConfiguration = runtimeSymbol("AWSClientConfiguration")
-        val AWSEndpoint = runtimeSymbol("AWSEndpoint", SwiftDeclaration.STRUCT)
-        val Partition = runtimeSymbol("Partition")
-        val ServiceEndpointMetadata = runtimeSymbol("ServiceEndpointMetadata")
-        val CredentialScope = runtimeSymbol("CredentialScope")
-        val UnknownAWSHTTPServiceError = runtimeSymbol("UnknownAWSHTTPServiceError", SwiftDeclaration.STRUCT)
-            .toInternalSPI("UnknownAWSHTTPServiceError")
-        val AWSServiceError = runtimeSymbol("AWSServiceError")
-        val RegionResolver = runtimeSymbol("RegionResolver")
-        val Sha256TreeHashMiddleware = runtimeSymbol("Sha256TreeHashMiddleware")
-        val AWSEndpointsRuleEngine = runtimeSymbol("AWSEndpointsRuleEngine")
-        val AWSEndpointsRequestContext = runtimeSymbol("AWSEndpointsRequestContext")
-        val AuthSchemeResolver = runtimeSymbol("AuthSchemeResolver")
-        val DefaultAuthSchemeResolver = runtimeSymbol("DefaultAuthSchemeResolver")
-        val AWSRetryErrorInfoProvider = runtimeSymbol("AWSRetryErrorInfoProvider")
-        val AWSRetryMode = runtimeSymbol("AWSRetryMode")
+        val UnknownAWSHTTPServiceError = runtimeSymbol("UnknownAWSHTTPServiceError", SwiftDeclaration.STRUCT, "UnknownAWSHTTPServiceError")
```

Comment:
This symbol has a SPI identifier added to it, which will be automatically added to the import.